### PR TITLE
Add hw folder, hw_2 done

### PR DESCRIPTION
**Pull request by dammitxoxo**

### CODE REVIEW

**Comment by AlekseyQty** [comment]
Bla bla

**Review by AlekseyQty — CHANGES_REQUESTED** [review]
Need to change something

File: saratkin_aleksey/hw_2/hw_2.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("I did it!")
```
Comment (AlekseyQty) [inline saratkin_aleksey/hw_2/hw_2.py:1]: This is wrong

**Review by AlekseyQty — COMMENTED** [review]
Need to fix this

File: saratkin_aleksey/hw_2/hw_2.py (hunk 1 above)
Comment (dammitxoxo) [reply saratkin_aleksey/hw_2/hw_2.py]: Add fix

**Pull request merged**